src/InitialConditions/initial_conditions.jl

```diff
@@ -673,7 +673,7 @@ end
 ##
 
 # By default, use the dycore's surface parametrization.
-surface_params(::InitialCondition, thermo_params) = nothing
+surface_params(::InitialCondition, thermo_params) = surface_params(Bomex(), thermo_params) 
```

Comment:
Why are we prescribing Bomex here?

Reply:
This is for CI cases where `turbconv_case` doesn't get set but EDMF is still run and needs some pseudo-default `InitialCondition` to get the `surface_params`. This will be removed again in the full surface PR.
For example this [case](https://buildkite.com/clima/climaatmos-ci/builds/9069#01879ad7-955c-4d04-bdb0-4ad8933da133):
```
julia --color=yes --project=examples examples/hybrid/driver.jl 
--job_id sphere_baroclinic_wave_rhoe_equilmoist_edmf --vert_diff true 
--surface_scheme bulk --moist dry --dt 1secs --t_end 20secs 
--FLOAT_TYPE Float64 --turbconv edmf --config sphere --z_stretch false 
--post_process false --apply_limiter false --dt_save_to_sol 10secs --dt_save_to_disk 10secs
```